_playbook/training.md

```diff
@@ -20,7 +20,7 @@ Some of our favorite, free accessibility courses include:
 
 Podcasts are another great way to learn about web accessibility—including [A11yRules.com](https://a11yrules.com/), [Interactive Accessibility](http://www.interactiveaccessibility.com/iap-podcasts) and [Human Potential At Work](https://www.ruhglobal.com/podcasthuman-potential-work/). While these resources are often available for free, the publishers usually depend on donations to support the ongoing education work. If you choose to listen, please consider contributing to a [Patreon campaign](https://www.patreon.com/steenhout) to support their wonderful work. Supporting these initiatives is a cost-effective way to increase the availability of new accessibility resources.
 
-There are also several high-quality accessibility guides available for free. The GDS has produced [accessibility guides](https://alphagov.github.io/accessibility-guidance/), as has the US government's [18F](https://accessibility.18f.gov/). Rather than creating a new guide from scratch, these resources are available to build upon and improve. CivicActions is looking to extend these to provide an [accessibility playbook](https://civicactions.github.io/accessibility/) for people interested in accessibility best practices.
```

Comment:
Maybe it should be "the US government has produced a guide with 18F"? Because 18F is an organization, not resources.

Reply:
@mikang good call. That makes more sense. Pushed a change